libcontainer/cgroups/devices/ebpf_linux.go

```diff
@@ -176,21 +176,18 @@ func loadAttachCgroupDeviceFilter(insts asm.Instructions, license string, dirFd
 	}
 
 	// If there is only one old program, we can just replace it directly.
-	var (
-		replaceProg *ebpf.Program
-		attachFlags uint32 = unix.BPF_F_ALLOW_MULTI
-	)
-	if useReplaceProg {
-		replaceProg = oldProgs[0]
-		attachFlags |= unix.BPF_F_REPLACE
-	}
-	err = link.RawAttachProgram(link.RawAttachProgramOptions{
+
+	attachProgramOptions := link.RawAttachProgramOptions{
 		Target:  dirFd,
 		Program: prog,
-		Replace: replaceProg,
 		Attach:  ebpf.AttachCGroupDevice,
-		Flags:   attachFlags,
-	})
+		Flags:   unix.BPF_F_ALLOW_MULTI,
+	}
+
+	if useReplaceProg {
+		attachProgramOptions.Anchor = link.ReplaceProgram(oldProgs[0])
```

Comment:
why not adding the flag too, as it was done before?

Reply:
They are now handled by the underlying code and are no longer explicitly allowed. See https://github.com/cilium/ebpf/blob/15d4f245e548eb00383aa88a24da934f0d26fc43/link/program.go#L34

Reply:
Thanks. For the next time, pointing this out in the commit/PR makes the review simpler.